lib/cylc/cylc_xdot.py

```diff
@@ -440,9 +440,9 @@ def get_graph(self, group_nodes=None, ungroup_nodes=None,
                 node.attr['shape'] = 'doubleoctagon'
                 # Detecting ghost families would involve analysing triggers
                 # in the suite's graphing.
-            elif self.is_ghost_task(name, point, cache=cache):
-                # Style ghost nodes.
-                style_ghost_node(node)
+            elif self.is_off_sequence(name, point, cache=cache):
+                node.attr['style'] = 'dotted'
+                node.attr['fontcolor'] = graph.graph_attr['fontcolor']
```

Comment:
We could keep the "ghost" effect by adding a couple of characters onto the end of the colour for transparency.

Reply:
That's true, I'll try this tomorrow.

Reply:
done